pkg/drivers/kic/oci/oci.go

```diff
@@ -175,11 +207,34 @@ func HostPortBinding(ociBinary string, ociID string, contPort int) (int, error)
 
 // ContainerIPs returns ipv4,ipv6, error of a container by their name
 func ContainerIPs(ociBinary string, name string) (string, string, error) {
+	if ociBinary == Podman {
+		return podomanConttainerIP(name)
+	}
+	return dockerContainerIP(name)
+}
+
+func podomanConttainerIP(name string) (string, string, error) {
```

Comment:
- Add comment that mentions it returns an IPv4/IPv6 pair.
- Fix function name spelling.